docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,145 @@
+{% from "userGuide/components/advanced.md" import slot_info_trigger %}
+
+## Annotations
+
+Annotation components allows you to easily add annotations over any image.
+
+### Introduction
+
+Annotation wrappers (`<annotate>`) are used in conjunction with annotation
+points (`<a-point>`).
+
+Annotation wrappers are used to hold the image and set the width and height of
+said image.
+
+Annotation points are used to define the position, text and style of each point
+within the image. In use, the annotation points are directly inserted between
+the annotate wrappers (`<annotate>...</annotate>`).
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Minimal Point -->
+  <a-point x="25%" y="25%" content="Lorem ipsum dolor sit amet" />
+  <!-- Customize Point Size (default size is 40px) -->
+  <a-point x="50%" y="25%" content="Lorem ipsum dolor sit amet"  size="60"/>
+  <!-- Customize Point Header (default is empty) -->
+  <a-point x="75%" y="25%" content="Lorem ipsum dolor sit amet"  header="Lorem ipsum"/>
+  <!-- Customize Point Color (default color is green) -->
+  <a-point x="33%" y="50%" content="Lorem ipsum dolor sit amet"  color="red"/>
+  <!-- Customize Point Opacity (default opacity is 0.3) -->
+  <a-point x="66%" y="50%" content="Lorem ipsum dolor sit amet"  opacity="0.7"/>
+  <!-- Customize Point Label (default is empty) -->
+  <a-point x="25%" y="75%" content="Lorem ipsum dolor sit amet" label="1"/>
+  <!-- Customize Text Color (default color is black) -->
+  <a-point x="50%" y="75%" content="Lorem ipsum dolor sit amet"  textColor="white" color="black" label="2" opacity="1"/>
+  <!-- Customize Font Size (default font size is 14) -->
+  <a-point x="75%" y="75%" content="Lorem ipsum dolor sit amet"  fontSize="30" label="3"/>
+</annotate>
+</variable>
+</include>
+
+**Using trigger for Annotate Point** <br>
+
+Similar to **[popovers](popups.md)**, annotation points also support different types of triggers
+for users with different needs.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="33%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="33%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+  <!-- Set Popover Placement (click)-->
+  <a-point x="25%" y="66%" content="Lorem ipsum dolor sit amet" placement="left"/>
+  <a-point x="50%" y="66%" content="Lorem ipsum dolor sit amet" placement="bottom"/>
+  <a-point x="75%" y="66%" content="Lorem ipsum dolor sit amet" placement="right"/>
+</annotate>
+</variable>
+</include>
+
+<br>
```

Comment:
I think can remove this br tag and the space

Reply:
this is done to ensure even spacing between two sections